hr_expense_sequence/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# © 2014 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
```

Comment:
`©` should to be replaced with `Copyright`, see sample https://github.com/OCA/maintainer-tools/tree/master/template/module/